File: samples/openapi3/server/petstore/kotlin-springboot/src/main/kotlin/org/openapitools/model/InlineObject1.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size

/**
 * 
 * @param additionalMetadata Additional data to pass to server
 * @param file file to upload
 */
data class InlineObject1(

    @JsonProperty("additionalMetadata") val additionalMetadata: kotlin.String? = null,

    @JsonProperty("file") val file: org.springframework.core.io.Resource? = null
) {

}

File: samples/client/petstore/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineObject1.scala

package org.openapitools.client.model

import java.io.File

case class InlineObject1 (
    /* Additional data to pass to server */
    _additionalMetadata: Option[String],
    /* file to upload */
    _file: Option[File]
)
object InlineObject1 {
    def toStringBody(var_additionalMetadata: Object, var_file: Object) =
        s"""
        | {
        | "additionalMetadata":$var_additionalMetadata,"file":$var_file
        | }
        """.stripMargin
}
